Sources/DreamsDelegate.swift

```diff
@@ -41,4 +41,10 @@ public protocol DreamsDelegate: AnyObject {
      Use this delegate callback to close or dismiss the dreams viewcontroller.
      */
     func handleExitRequest()
+    
+    /**
+        Called when a transfer requires a consent.
+     */
+
+    func handleDreamsTransferConsentRequested(requestId: String, consentId: String, completion: @escaping (Result<TransferConsentSuccess, TransferConsentError>) -> Void)
```

Comment:
why handle**Dreams**TransferConsentRequested ?

Reply:
taking a look at the whole file all of the functions except the one you see in the preview are named with dreams:P
https://github.com/dreamstechnology/dreams-ios-sdk/blob/465bb9a2a5772154f090a5d91424c578abcfaafb/Sources/DreamsDelegate.swift

Reply:
Ha! True. Yeah, let's keep it consistent then 👍